stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -589,18 +589,27 @@ end
 #! via the advice stack.
 #! The signature is valid if and only if the procedure returns.
 #!
-#! Input: [PK, MSG, NONCE1, NONCE1, ...]
+#! Input: [PK, MSG, ...]
 #! Output: [...]
 #!
 #! Cycles: ~ 92029
 export.verify.1665
 
-    # 1) Load the public key polynomial h and the signature polynomial s2 and the product of
+    # 1) Generate a Falcon signature using the secret key associated to PK on message MSG.
+    adv.sig_to_stack.rpo_falcon512
```

Comment:
Similar to the previous comment, I think this should be `adv.push_sig.rpo_falcon512`.

Reply:
Corrected!